scripts/generate-module-icons.py

```
@@ -0,0 +1,100 @@
+  <path d="M74 68 V82 M68 76 L74 83 L80 76" fill="none" stroke="{GOLD}" stroke-width="6" stroke-linecap="round" stroke-linejoin="round"/>''')
+
+# Pwa: phone with install arrow.
+ICONS["BarakoCMS.Pwa"] = ("BarakoCMS.Pwa", "#009688", f'''  <rect x="42" y="22" width="44" height="84" rx="12" fill="{WHITE}"/>
+  <circle cx="64" cy="96" r="4" fill="#009688"/>
+  <path d="M64 44 V70 M53 60 L64 71 L75 60" fill="none" stroke="{GOLD}" stroke-width="8" stroke-linecap="round" stroke-linejoin="round"/>''')
+
+base = os.path.expanduser("~/repos/barakoCMS")
+for mod, (label, bg, glyph) in ICONS.items():
+    path = os.path.join(base, mod, "assets", "icon.svg")
+    if not os.path.isdir(os.path.dirname(path)):
+        print("SKIP missing dir:", path); continue
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)